### PR TITLE
Update ICGC gitops.json

### PR DESCRIPTION
**Pull request by FayBooker2**
Per request of Trevar Simmons in CTDS Slack.  https://cdis.slack.com/archives/CP24V8MUG/p1733241956938919

### Environments
 icgc.bionimbus.org

### Description of changes
Added banner to portal/gitops.json announcing maintenance on Dec 6

### CODE REVIEW

File: icgc.bionimbus.org/portal/gitops.json [hunk 1]
```diff
@@ -18,7 +18,14 @@
   },
   "components": {
     "appName": "Bionimbus PDC for ICGC",
-    "index": {
+      "banner": [
+        {
+          "type": "warning",
+          "message": "The ICGC Site will undergoing maintenance on Friday, December 6, 2024, 8am-5pm CST.  Users may experience service interruptions during this time.",
```
Comment (trevars) [inline icgc.bionimbus.org/portal/gitops.json:24]: Instead of calling it the ICGC site, can we modify to "Bionimbus Protected Data Cloud"? Thanks!

**Review by trevars — DISMISSED** [review]
Minor request to change the name.

File: icgc.bionimbus.org/portal/gitops.json (hunk 1 above)
Comment (FayBooker2) [reply icgc.bionimbus.org/portal/gitops.json]: Sure.

**Review by beamcb — APPROVED** [review]

**Review by FayBooker2 — COMMENTED** [review]
Corrected

**Pull request merged**